src/layer/marker/Marker.js

```diff
@@ -123,6 +127,30 @@ L.Marker.include({
         this._resetZIndex();
     },
 
+    /**
+     * @since leaflet@v1.8
+     * @see https://github.com/Leaflet/Leaflet/commit/4f639a85efffa49c3e64a07dc0b6f5aa73f13449
+     */
+    _panOnFocus: function () {
+        /**
+         * Temporary disable this for fix: https://github.com/Raruto/leaflet-rotate/issues/18
+         * 
+         * @TODO restore it and support for `L.Marker::autoPanOnFocus` option
+         */
+
+        // var map = this._map;
+        // if (!map) { return; }
+
+        // var iconOpts = this.options.icon.options;
+        // var size = iconOpts.iconSize ? L.point(iconOpts.iconSize) : L.point(0, 0);
+        // var anchor = iconOpts.iconAnchor ? L.point(iconOpts.iconAnchor) : L.point(0, 0);
+
+        // map.panInside(this._latlng, {
+        //     paddingTopLeft: anchor,
+        //     paddingBottomRight: size.subtract(anchor)
+        // });
+    },
+
```

Comment:
@siimaoo the reason behind that bug should be this new feature (added by leaflet v1.8).

For the moment I have disabled it, could you help me in trying to re-enable it?

If you take a quick look to the rest of this plugin code in general it's just about "adding" a rotation function among those already present in this library, the difficult thing is enough time right to find the right one..

https://leafletjs.com/reference#marker-autopanonfocus

Ref: https://github.com/Raruto/leaflet-rotate/issues/18

Reply:
tonight (gmt -3) I'll take a look at this